packages/devextreme-react/package.json

```diff
@@ -32,8 +32,8 @@
     "prop-types": "^15.8.1"
   },
   "devDependencies": {
-    "@testing-library/react": "^13.3.0",
-    "@testing-library/user-event": "^13.0.1",
+    "@testing-library/react": "^14.2.1",
```

Comment:
I updated testing library because with the new version, I was able to get rid of all the warnings during tests.